spec/consensus/proposer-based-timestamp/tla/HowToRun.md

```diff
@@ -0,0 +1,27 @@
+# How to run tests
```

Comment:
this should probably be a readme.md file, (though our filenaming convention is spotty, we seem to typically use _ or - rather than camel case in file names

Reply:
Agreed, I will suggest this to Jure (the author).